docs/_docs/ssl-tls.adoc

```diff
@@ -0,0 +1,193 @@
+   .trustManager(getTrustManagerFactory());
+}
+----
+
+=== Server-side Configuration
+
+[source,java]
+----
+SslContextBuilder builder = SslContextBuilder.forServer(certChainInput, keyInput)
+    .ciphers(getCiphers(), getCiphersFilter())
+    .sessionTimeout(serverSslConfig.getSessionTimeout())
+    .sslProvider(sslProvider);                 
+.trustManager(trustedCerts.toArray(new X509Certificate[0]))
+    .clientAuth(serverSslConfig.getClientAuth());
+----
+
+View the link:https://github.com/devsunny/netty-ssl-example/blob/master/src/main/java/com/asksunny/ssl/SecureSocketSslContextFactory.java[Netty SSL configuration].
+
+Introduce the client configuration on the Apache Ignite 3.x server side and map it to Netty Security Context: 
+
+[source,json]
+----
+"clientConnector": {
+    "ssl": {
+        "enabled": false,
+        "clientAuth": "none",
+        "keyStore": {
+            "type": "PKCS12",
+            "path": "must not be empty",
+            "password": "may be empty"
+        },
+        "trustStore": {
+            "type": "PKCS12",
+            "path": "must not be empty",
+            "password": "may be empty"
+        }
+    }
+}
+----
+
+If you have enabled SSL for `clientConnector`, set the corresponding properties in link:https://github.com/apache/ignite-3/blob/be6c8b290894dbd6f88eaaa2a2aafc3eff300855/modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc/ConnectionProperties.java[ConnectionProperties]. 
+
+== Platform Clients
+
+=== .NET
+
+Add the `IgniteClientConfiguration.SslStreamFactory` property of type `ISslStreamFactory`.
+
+Provide a link:https://github.com/apache/ignite/blob/66f43a4bee163aadb3ad731f6eb9a6dfde9faa73/modules/platforms/dotnet/Apache.Ignite.Core/Client/SslStreamFactory.cs[predefined implementation].
+
+Use the base class library `SslStream`.
+
+Basic usage without client authorization:
+
+[source,csharp]
+----
+var cfg = new IgniteClientConfiguration { SslStreamFactory = new() }
+----
+
+=== C++
+
+TBD
+//Do we need to include this section and say that we expect the C++ client SSL to be done soon? Or just remove this section for now?
+
+== CLI Configuration
+
+The CLI, uses OkHTTP to communicate with Apache Ignite 3.x. To enable SSL, create `SSLSocketFactory` and pass it to the `OkHttpClient` builder:
+
+[source,java]
+----
+X509TrustManager trustManager;
+SSLSocketFactory sslSocketFactory;
+try {
+    trustManager = trustManagerForCertificates(trustedCertificatesInputStream());
+    SSLContext sslContext = SSLContext.getInstance("TLS");
+    sslContext.init(null, new TrustManager[] { trustManager }, null);
+    sslSocketFactory = sslContext.getSocketFactory();
+} 
+catch (GeneralSecurityException e) {
+    throw new RuntimeException(e);
+}
+
+client = new OkHttpClient.Builder()
+    .sslSocketFactory(sslSocketFactory, trustManager)
+    .build();
+----
+
+You can enable SSL on the CLI side using the `cli config set` command:
+
+[source,shell]
+----
+cli.trust-store.type=
```

Comment:
This is not the whole command that the user would execute. Could you please type here commands like:

```bash
cli config set cli.trust-store.type=<type>
cli config set cli.trust-store.path=<path>
cli config set cli.trust-store.password=<password>